tests/test_runs/test_pipelines/predict.py

```diff
@@ -11,7 +11,7 @@
 def main_hydra(cfg: DictConfig) -> None:
     cfg = dictconfig_to_dict(cfg)
     download_mock_dataset(MOCK_DATASET_PATH)
-    cfg["data_dir"] = MOCK_DATASET_PATH
+    cfg["data_dir"] = str(MOCK_DATASET_PATH)
```

Comment:
the idea here and in other similar changes below is to have the same types as in real runs of our pipelines